improver/uv_index.py

```diff
@@ -76,9 +69,9 @@ def calculate_uv_index(
         A cube of the calculated UV index.
 
     Raises:
-        ValueError: If uv_upward is not named correctly.
         ValueError: If uv_downward is not named correctly.
-        ValueError: If units do not match.
+        ValueError: If uv_downward contains values that are negative or
+        not a number.
 
     References:
         Turner, E.C, Manners, J. Morcette, C. J, O'Hagan, J. B,
```

Comment:
Not your fault, but please correct this spelling:
```suggestion
        Turner, E.C, Manners, J. Morcrette, C. J, O'Hagan, J. B,
```